pyworkflow/project/project.py

```diff
@@ -634,7 +634,9 @@ def launchProtocol(self, protocol, wait=False, scheduled=False,
             pwutils.path.copyFile(self.dbPath, protocol.getDbPath())
 
         # Launch the protocol, the jobId should be set after this call
-        pwprot.launch(protocol, wait)
+        jobId = pwprot.launch(protocol, wait)
+        if jobId is None or jobId == -1:
```

Comment:
Should we use UNKNOWN_JOBID instead of -1 ?

Reply:
it's not defined in constants, just in one file. Do you want to move it to constants?

Reply:
I do not see it necessary if used within the same file, for me is fine to be at the top. But I'm ok if you want to move it there.